Exercise_1/day1/Program15.java

```diff
@@ -0,0 +1,78 @@
+package day1;
+
+public class Program15 {
+  public static void main(String[] agrs) {
+	  System.out.println("Prime Numbers Between 1 to 100 :");
+	  primeNumber();
+	  System.out.println();
+	  Program15b pA = new Program15b();
+	  int number = 28;
+	  System.out.println();
+	  System.out.println("check wether given number is Armstrong number , Palindrone, Perfect Number ");
+	  System.out.println();
+	  pA.armstrongNumber(number);
+	  pA.palindromeNumber(number);
+	  pA.perfectNumber(number);
+  }
+  static void primeNumber()
+  {
+	  System.out.println();
```

Comment:
name the variables more meaning fully instead of using single letters

Reply:
Updated